### PR TITLE
add testing & ci slides

### PR DESCRIPTION
**Pull request by tschuy**
Also, can I get perms on the main repo?

### CODE REVIEW

**Comment by lucywyman** [comment]
@jordane ^^

File: source/bootcamp_content/testing-and-ci.rst [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+What are tests?
+---------------
+
+Tests are a way to make sure your code does what you think your code does.
+
+They can involve testing input to your software to make sure it handles both
+normal and edge cases properly, that it will work in its intended environment,
+etc.
+
+.. figure:: static/qaengineer.png
```
Comment (lucywyman) [inline source/bootcamp_content/testing-and-ci.rst]: These need to be /static/blah.png

File: source/bootcamp_content/testing-and-ci.rst [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+--------------
+
+This is the last type of testing you'll do in building a system.
+It tests how the entire sytem works from start to finish, and verifies
+that it meets all of the requirements.
+
+System testing pretends the system is a **black box**. It ignores how the code
+actually works, and focuses on how the platform as a whole works.
+
+.. figure:: static/black-box.png
```
Comment (lucywyman) [inline source/bootcamp_content/testing-and-ci.rst]: This is too big for my screen :/

File: source/bootcamp_content/testing-and-ci.rst [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+* Travis CI
+* Jenkins
+
+Travis CI
+---------
+
+Integrates into Github, allowing your tests to be run each time you push code.
+
+.. figure:: static/travis.png
+    :align: center
```
Comment (lucywyman) [inline source/bootcamp_content/testing-and-ci.rst]: Seems like an abrupt ending?
